frontend/tests/robot/RF-tests/linkpage/suiteKeywords.resource

```diff
@@ -0,0 +1,16 @@
+*** Settings ***
+Documentation    This file contains common keywords and variables used with test suites.
+
+*** Keywords ***
+Go To Links Page
+    [Documentation]    Clicks on the button that takes the user to the linkpage.
+    Click Button    Linkkejä
```

Comment:
tähä alle vois lisätä 'Page Should Contain ...' mikä se otsikko nyt sil sivul onkaa, et varmistaa et toi Linkkejä button vie oikeelle sivulle